blog/templates/blog/base.html

```diff
@@ -9,15 +9,20 @@
     </head>
     <body>
         <header class="page-header">
-          <div class="container">
-              <h1><a href="/">Django Girls Blog</a></h1>
-          </div>
+            <div class="container">
+            {% if user.is_authenticated %}
+                <a href="{% url 'post_new' %}" class="top-menu">
+                    {% include './icons/file-earmark-plus.svg' %}
+                </a>
+            {% endif %}
+                <h1><a href="/">Django Girls Blog</a></h1>
+            </div>
         </header>
-        <main class="container">
+        <main class="content container">
             <div class="row">
                 <div class="col">
-                {% block content %}
-                {% endblock %}
+                    {% block content %}
+                    {% endblock %}
```

Comment:
there's no need to indent this

Reply:
I will change indent in the separate PR.

Reply:
why?

Reply:
I fixed the extra indents.

Reply:
resolved.